group_by.py

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Don't use abbreviations in variable names -
https://softwareengineering.stackexchange.com/questions/72593/how-to-abbreviate-variable-names

Reply:
func variable name is not good as well